recipes/pyloh/build.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-$PYTHON setup.py install
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
`$PYTHON setup.py install` is actually fine since the package does not use `setuptools`.